nxrm-aws-resiliency/templates/workdir-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-{{ .Chart.Version }}-{{ .Release.Name }}-create-nexus-workdir-config
```

Comment:
as far as I know, this name must be no more than 63 characters. 
So `nxrm-aws-resiliency-45.1.0-<Release.Name>-create-nexus-workdir-config-?????????` contains more than 63 symbols event without `Release.Name`.

Reply:
https://github.com/sonatype/nxrm3-helm-repository/pull/41/commits/35fb1119fb025a37ef8acd8ed029b7d3fa06d55e